### PR TITLE
Feature/Implementation-of-salary-slip-show-and-print-functionality

### PR DESCRIPTION
**Pull request by JoharAlam**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -39,6 +39,19 @@
 
 //Route::Post('/newSlackbot', 'AttendanceController@newSlackbot')->name('newSlackbot');
 
+Route::group(['middleware' => 'auth'], function () {
```
Comment (awaidanjum) [inline routes/web.php:42]: @JoharAlam You don't have to use the same middleware group function a second time, you can add these routes to already created Auth route group.

File: app/Http/Controllers/SalarySlipController.php [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+                    $approvedDeduction = 0;
+                }
+            } elseif ($employee->emloyment_status == 'probation') {
+                $approvedDeduction = $approvedCount[$employee->id];
+            }
+            $absentDeduction[$employee->id] = ($employee->basic_salary / $workingDays) * $AbsentCount[$employee->id];
+            $netPayables[$employee->id] = round(($employee->basic_salary - ($employeeUnApprovedLeaves[$employee->id] * 2) - $approvedDeduction - $absentDeduction[$employee->id]) + ($employee->bonus));
+
+            // $unApprovedCount[$employee->id]=$employeeUnApprovedLeaves[$employee->id];
+            // $approvedCount[$employee->id]=$employeeApprovedLeaves[$employee->id];
```
Comment (awaidanjum) [inline app/Http/Controllers/SalarySlipController.php:253]: @JoharAlam 
Please remove commented code.

Comment (JoharAlam) [reply app/Http/Controllers/SalarySlipController.php:253]: Ok will do it

File: routes/web.php (hunk 1 above)
Comment (JoharAlam) [reply routes/web.php:42]: @awaidanjum I will fix it

**Comment by JoharAlam** [comment]
@faizan-glowlogix bhai Please Review

**Pull request merged**